editor/src/core/model/element-metadata-utils.ts

```diff
@@ -1099,7 +1061,7 @@ export const MetadataUtils = {
   getJSXElementName(
     path: TemplatePath,
     components: Array<UtopiaJSXComponent>,
-    metadata: ComponentMetadata[],
+    metadata: ComponentMetadata[], // TODO maybe we could remove metadata as a dependency from here if we change findSceneByTemplatePath
```

Comment:
Have you investigated whether or not these are possible?

Reply:
not yet, but I think it should be fine, just didn't want to bloat this PR